newEntry.php

```diff
@@ -1 +1,45 @@
 <?php
+session_start();
+?>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <title>My Vinyl Collection</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="normalize.css" type="text/css" rel="stylesheet">
+    <link href="stylesheet.css" type="text/css" rel="stylesheet">
+</head>
+<body>
+<div class="title_bar">
+    <div class="logo"><img src="logo1.png" alt="Jenny Logo"></div>
+    <h1>MY VINYL COLLECTION</h1>
+    <div></div>
```

Comment:
empty div?

Reply:
to balance out the  nav bar so it has three items on it, and they all stay in the right position (ie. in the main page there is a button in that position, and "space-evenly" works nicely. But on this page there were only two items so I needed an invisible thing there to balance the three. maybe there's a better way to do it?

Reply:
ah ok, might be fine then